index.html

```diff
@@ -55,8 +55,45 @@
       }
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-    <script>
-      // Put some code in here!
+        <script>
+    var countDown = 1234;
+    //test//
+    // $("#counter").val(countDown);
+
+    function Start(){
+      
+      console.log("I got clicked");
+      $("#counter").val(countDown);
+      setInterval(Counter(), 1000);
```

Comment:
Did you want setInterval to call this function every 1000ms or did you want it to call the result of this function every 1000ms?  This code executes Counter() and passes the result to setInterval - asking setInterval to run whatever is returned from Counter().  We rarely do this - it requires a function that returns a function.  You almost always simply want to pass the function itself: `setInterval(Counter, 1000)`.  Notice... no parens.